### PR TITLE
Added recipe for cliff

### PR DESCRIPTION
**Pull request by pmlandwehr**
Getting back on the path of an openstack client build... `cliff` is another component of the system.

Pinging @dhellmann in case he'd like to be added as a maintainer to the `cliff` builder on [conda-forge](http://conda-forge.github.io), a library of community-built [`conda`](http://conda.pydata.org) packages…

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Forcing a rebuild on the assumptions that windows missed the final `cmd2

**Comment by pmlandwehr** [comment]
(This build seems to be stuck, so am restarting)

**Comment by patricksnape** [comment]
Manually restarted Travis - now passes

File: recipes/cliff/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  name: cliff
+  version: {{ version }}
+
+source:
+  fn: cliff-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cliff/cliff-{{ version }}.tar.gz
+  sha256: 0c3d100cbf47d168bdca70acf890f1fc024abdadac13ede1336b0f8075403dea
+
+build:
+  # skip: True  # [py35]
```
Comment (ocefpaf) [inline recipes/cliff/meta.yaml]: Please remove this commented line.

File: recipes/cliff/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=1.6
```
Comment (ocefpaf) [inline recipes/cliff/meta.yaml]: Is `pbr` needed at run time too?

File: recipes/cliff/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  imports:
+    - cliff
+    - cliff.app
+    - cliff.commandmanager
+    - cliff.command
+    - cliff.interactive
+
+about:
+  home: https://github.com/openstack/cliff
+  license: Apache Software License
```
Comment (ocefpaf) [inline recipes/cliff/meta.yaml]: Please add the license version.

File: recipes/cliff/meta.yaml (hunk 2 above)
Comment (dhellmann) [inline recipes/cliff/meta.yaml]: It should only be needed to build an sdist or wheel.

Comment (ocefpaf) [inline recipes/cliff/meta.yaml]: Thanks for the clarification!

File: recipes/cliff/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/cliff/meta.yaml]: Looks like Apache 2.0.

**Comment by jakirkham** [comment]
Just FYI staged-recipes is blocked up because of Travis CI API issues.

**Pull request merged**